packages/notification-services-controller/package.json

```diff
@@ -45,7 +45,7 @@
     "@metamask/base-controller": "^6.0.0",
     "@metamask/controller-utils": "^11.0.0",
     "@metamask/keyring-controller": "^17.1.0",
-    "@metamask/profile-sync-controller": "^0.0.0",
+    "@metamask/profile-sync-controller": "^0.1.0",
```

Comment:
This fails the publish preview since this controller hasn't been released yet.
Maybe we could split this into 2 releases?